core/controller/src/main/scala/whisk/core/controller/RestAPIs.scala

```diff
@@ -84,12 +83,12 @@ abstract protected[controller] class RestAPIVersion(
 protected[controller] object RestAPIVersion_v1 {
     def requiredProperties =
         WhiskConfig.whiskVersion ++
-            WhiskServices.requiredProperties ++
+            WhiskAuthStore.requiredProperties ++
```

Comment:
consolidating the required properties is ok - since the api unit tests use the common controller test trait which unifies the required properties also.